tasks/main.yml

```diff
@@ -1,25 +1,32 @@
 # SPDX-License-Identifier: MIT
 ---
+
 - name: Set platform/version specific variables
   include_tasks: tasks/set_vars.yml
 
-# Examples of some tasks:
 - name: Ensure required packages are installed
   package:
-    name: "{{ __template_packages }}"
+    name: "{{ __keylime_server_packages }}"
     state: present
 
 - name: Ensure required services are enabled and started
```

Comment:
Is it fine that the role starts and enables services that early in the playbook? One scenario where this can lead to an unwanted failure that comes to my mind is when a user has some failing configuration present and wants to fix it with the role, but the role starts the services prior to applying a proper configuration and fails. I do not know if this task would fail in the case of a failing configuration though.

Reply:
Let me do some testing for this case.

Reply:
any updates?